tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -336,7 +363,13 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
                                               new[] { ContextPropagationHeaderStyle.Datadog, ContextPropagationHeaderStyle.W3CTraceContext },
                                               $"{ContextPropagationHeaderStyle.Datadog},{ContextPropagationHeaderStyle.W3CTraceContext}"),
                                           validator: styles => styles is { Length: > 0 }, // invalid individual values are rejected later
-                                          converter: style => TrimSplitString(style, commaSeparator));
+                                          converter: style => TrimSplitString(style, commaSeparator),
+                                          openTelemetryKey: "OTEL_PROPAGATORS",
```

Comment:
Should this be moved to `ConfigurationKeys.OpenTelemetry`?

Reply:
Good spot! Fixed in 8409fd6f546a39f6275461d50a36e43bb13455e8